servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -789,6 +789,19 @@ public final Completable afterOnSubscribe(Consumer<Cancellable> onSubscribe) {
         return afterSubscriber(doOnSubscribeSupplier(onSubscribe));
     }
 
+    /**
+     * Invokes the {@code onSubscribe} {@link Consumer} argument <strong>when</strong>
```

Comment:
(Applicable to all methods added in this PR) 

Since `when` does not have any semantic meaning as compared to before/after, we do not need to enclose it in `<strong>` tags.

Reply:
Done!